osu.Game/Graphics/UserInterface/ExternalLinkButton.cs

```diff
@@ -44,6 +50,23 @@ public ExternalLinkButton(string link = null)
             };
         }
 
+        private class CopyUrlToast : Toast
+        {
+            public CopyUrlToast(LocalisableString value)
+                : base(UserInterfaceStrings.GeneralHeader, value, "")
+            {
+            }
+        }
+
+        private void copyUrl()
+        {
+            if (Link != null)
+            {
+                host.GetClipboard()?.SetText(Link);
+                onScreenDisplay?.Display(new CopyUrlToast(ToastStrings.CopiedUrl));
+            }
```

Comment:
Link is guaranteed not to be null from `ContextMenuItems`.
```suggestion
            host.GetClipboard()?.SetText(Link);
            onScreenDisplay?.Display(new CopyUrlToast(ToastStrings.CopiedUrl));
```